doc/security.md

```diff
@@ -41,11 +41,14 @@ The following sample code authenticates as an installation using [lcobucci/jwt 4
 to generate a JSON Web Token (JWT).
 
 ```php
+use Github\HttpClient\Builder;
 use Lcobucci\JWT\Configuration;
 use Lcobucci\JWT\Encoding\ChainedFormatter;
 use Lcobucci\JWT\Signer\Key\LocalFileReference;
 use Lcobucci\JWT\Signer\Rsa\Sha256;
 
+$builder = new Builder(new GuzzleHttp\Client());
```

Comment:
not recommended to pass a client like this

Reply:
recommended way is to call it without argument?

Reply:
removed